odl/discr/discretization.py

```diff
@@ -316,6 +316,26 @@ def __setitem__(self, indices, values):
         else:
             self.ntuple.__setitem__(indices, values)
 
+    @property
+    def restriction(self):
+        """The restriction operator associated with this vector.
+
+        See Also
+        --------
+        RawDiscretization.restriction : For full description
+        """
+        return self.space.restriction(self.ntuple)
```

Comment:
How does that make sense? According to the docstring you want to return the operator. It cannot be applied to the `ntuple`.